### PR TITLE
reproducers for gh-5533 and gh-5386

### PR DESCRIPTION
**Pull request by jsafarik**
//test: `@integrations-odata-read-update`

### CODE REVIEW

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/odata.feature [hunk 1]
```diff
@@ -153,4 +153,67 @@ Feature: OData Connector
...
+    Then Integration "<name>" is present in integrations list
+    And wait until integration "<name>" gets into "Running" state
+
+    #Validate logs output
+    When sleep for "5000" ms
+    Then validate that OData service contains entity with "Description":"<value>" property:value pair in "Products" collection
+
+    Examples:
+      | name            | key_predicate | filter | split | value      |
+      # gh-5386
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/odata.feature]: Turns out you can actually tag individual examples like so:
```
    @gh-5386
    Examples:
      | name            | key_predicate | filter | split | value      |
      | wrong_datashape | 2             | false  | false | 1UMTS PDA  |
      | calling_null    |               | false  | false | Flat Basic |

    @gh-5533
    Examples:
      | name            | key_predicate | filter | split | value      |
      | NPE             | 2             | true  | true  | 1UMTS PDA  |
```

That will make sure they are correctly skipped and reported.

Comment (jsafarik) [reply ui-tests/src/test/resources/features/integrations/odata.feature]: Oh, ok, didn't even know there could be 2 example sets in 1 scenario outline. Thanks for the tip, should be fixed now (hopefully).

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**